packages/calling-stateful-client/src/PPTLiveSubscriber.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(ppt-live) */
+import { Features, PPTLiveCallFeature } from '@azure/communication-calling';
+/* @conditional-compile-remove(ppt-live) */
+import { CallCommon } from './BetaToStableTypes';
+/* @conditional-compile-remove(ppt-live) */
+import { CallContext } from './CallContext';
+/* @conditional-compile-remove(ppt-live) */
+import { CallIdRef } from './CallIdRef';
+
+/* @conditional-compile-remove(ppt-live) */
+/**
+ * @private
+ */
+export class PPTLiveSubscriber {
+  private _callIdRef: CallIdRef;
+  private _context: CallContext;
+  private _call: CallCommon;
+  private _pptLive: PPTLiveCallFeature;
+
+  constructor(callIdRef: CallIdRef, context: CallContext, call: CallCommon) {
+    this._callIdRef = callIdRef;
+    this._context = context;
+    this._pptLive = call.feature(Features.PPTLive);
+    this._call = call;
+
+    this.subscribe();
+  }
+
+  private subscribe = (): void => {
+    this._pptLive.on('isActiveChanged', this.isAvailableChanged);
+  };
+
+  public unsubscribe = (): void => {
+    this._pptLive.off('isActiveChanged', this.isAvailableChanged);
+  };
+
+  private isAvailableChanged = (): void => {
+    this._context.setCallPPTLiveActive(this._callIdRef.callId, this._pptLive.isActive);
+    this.checkAndUpdatePPTLiveParticipant();
+  };
+
+  private checkAndUpdatePPTLiveParticipant = async (): Promise<void> => {
+    if (this._pptLive.isActive) {
+      if (this._call.isScreenSharingOn) {
+        await this._call.stopScreenSharing();
```

Comment:
why does the stateful layer here need to stop screensharing? Does the SDK support both screensharing and pptLive being active at the same time? If so we should move this logic to the component/composite layer as the stateful client should be agnostic of the application (some applications may want to show both).
If the SDK doesn't support both, the SDK layer should be stopping the screensharing as every developer would have to call this

Reply:
The SDK does not have this logic right now, I will remove this logic and back to SDK to refactor.

Thanks for suggestion.